apps/livreGUI/Controller.cpp

```diff
@@ -21,238 +21,102 @@
  */
```

Comment:
It's just the current way we used it. I have also no objections to change if there is a need for it. In the next PR I have a ZeroBuf object for the renderer parameters which enhances the controller accordingly.